Cargo.lock

```diff
@@ -209,6 +228,30 @@ dependencies = [
  "slab",
 ]
 
+[[package]]
```

Comment:
AFAIK there is a bug in libp2p with async-std 1.6.0

CC @tomaka

Reply:
We don't know if it's a bug in async-std or in libp2p. Libp2p is now depending on `async-std <1.6`. It might be wise to not upgrade async-std.